dashboard/src/components/ConfirmDialog/index.tsx

```diff
@@ -39,19 +40,23 @@ class ConfirmDialog extends React.Component<IConfirmDialogProps, IConfirmDialogS
           {this.state.error && (
             <div className="padding-big margin-b-big bg-action">{this.state.error}</div>
           )}
-          <div>Are you sure you want to delete this?</div>
-          <div>
-            <button className="button" onClick={this.props.closeModal}>
-              Cancel
-            </button>
-            <button
-              className="button button-primary button-danger"
-              type="submit"
-              onClick={this.props.onConfirm}
-            >
-              Delete
-            </button>
-          </div>
+          {this.props.loading === true ? (
```

Comment:
this is needed to wait until the Tiller release is deleted, if not, after clicking "Delete" it goes directly to the Applications view and the delete application is still there.